pkgs/by-name/ve/vep/package.nix

```diff
@@ -0,0 +1,126 @@
+# Instructions:http://www.ensembl.org/info/docs/api/api_installation.html
+# Do not use https://github.com/Ensembl/ensembl-vep/archive/release/${version}.zip
+# We cannot use INSTALL.pl but it’s not that bad:install the dependencies and copies the .pm files should be ok
+{
+  lib,
+  perlPackages,
+  pkgs,
+  stdenv,
+  fetchFromGitHub,
+  fetchzip,
+  git,
+  perl,
+  unzip,
+  makeWrapper,
+  which,
+  zlib,
+}:
+
+let
+  version = "110";
+  customInstallPhase = ''
+    mkdir -p $out/${perl.libPrefix}/${perl.version}/
+    tests=$(find modules/ -mindepth 1 -maxdepth 1 -type d | grep -v t)
+    cp -r $tests $out/${perl.libPrefix}/${perl.version}/
+  '';
+
+  ensemblGit =
+    name: sha256:
+    # Copy modules directly
+    stdenv.mkDerivation {
+      inherit name version;
+      # Not available through fetchFromGitHub
+      src = fetchzip {
+        inherit sha256 version;
+        url = "https://github.com/Ensembl/${name}/archive/release/${version}.zip";
+      };
+      installPhase = customInstallPhase;
+    };
+
+  vepPlugins = fetchFromGitHub {
+    owner = "Ensembl";
+    repo = "VEP_plugins";
+    rev = "8f271c4848338dc7d504881ff71fdf2892c3d096";
+    sha256 = "sha256-LbaXwLFDP3m1QhRHwO9uh36BEFHE2NzL4xdxTb7/S5Q=";
+  };
+
+  # Install ensembl-xs, faster run using re-implementation in C of some of the Perl subroutines
+  ensembl-xs = perlPackages.buildPerlPackage rec {
+    pname = "ensembl-xs";
+    version = "2.3.2";
+    src = fetchFromGitHub {
+      inherit version;
+      owner = "Ensembl";
+      repo = "ensembl-xs";
+      rev = version;
+      sha256 = "1qqnski532f4bz32wxbqd9w1sz40rjh81ipp9p02k3rlaf1gp1fa";
+    };
+    # PREFIX is important
+    configurePhase = ''
+      perl Makefile.PL PREFIX=$out INSTALLDIRS=site
+    '';
+    # Test do not work -- wrong include path
+    doCheck = false;
+  };
+
+  # it contains compiled versions of certain key subroutines used in VEP
+  ensembl = ensemblGit "ensembl" "sha256-labyyqp+FRLIN9y4koePkX5fUls8HbCJlgYHUA0JR/0=";
+  ensembl-io = ensemblGit "ensembl-io" "sha256-r3RvN5U2kcyghoKM0XuiBRe54t1U4FaZ0QEeYIFiG0w=";
+  ensembl-variation = ensemblGit "ensembl-variation" "sha256-UjrLHF9EqI+Mp+SZR4sLNZUCGiA/UYhoFWtpwiKF8tM=";
+  ensembl-funcgen = ensemblGit "ensembl-funcgen" "sha256-a9hxLBoXJsF5JWuRdpyOac1u033M8ivEjEQecuncghs=";
+in
+perlPackages.buildPerlModule rec {
+  pname = "ensembl-vep";
+  version = "110";
```

Comment:
This should match the version string used by upstream, which looks like it's 110.0.